src/app/course-page/components/course-page/course-page.component.ts

```diff
@@ -56,9 +80,13 @@ export class CoursePageComponent implements OnInit {
     console.log('OnInit CoursePage Component', this.counter);
   }
 
-  public onRootDelete(item: CourseItem): void {
+  public onItemDelete(item: CourseItem): void {
     this.courseItems = this.courseItems.filter((course: CourseItem) => course.id !== item.id);
     console.log(item.id);
   }
 
+  onValueInParentChanged(value: string) {
```

Comment:
I do not recommend using naming with 'Parent' included, please, adjust names.

Reply:
Renamed names